skill-tree/src/test/java/com/RDS/skilltree/integration/EndorsementsIntegrationTests.java

```diff
@@ -490,4 +516,65 @@ public void itShouldReturn401OnEndorsementSearchWithoutCookie() {
                         equalTo(
                                 "The access token provided is expired, revoked, malformed, or invalid for other reasons."));
     }
+
+    @Test
+    @Disabled
+    @DisplayName(
+            "Return 200, when request is made using super user cookie and status is APPROVED/REJECTED")
+    public void
+            itShouldReturn200OnUpdateEndorsementStatusWithSuperUserCookieAndAcceptOrRejectEndorsementStatus() {
+        String endorsementId = createEndorsementModel(true);
+        Response response =
+                given()
+                        .cookies(RestAPIHelper.getSuperUserCookie())
+                        .queryParam("status", EndorsementStatus.APPROVED.name())
+                        .patch("/v1/endorsements/{id}", endorsementId);
+
+        response
+                .then()
+                .statusCode(200)
+                .body("data", equalTo(null))
+                .body("message", equalTo("Successfully updated endorsement status"));
+    }
+
+    @Test
+    @Disabled
+    @DisplayName(
+            "Return 401, when request is made without using super user cookie and status is APPROVED/REJECTED")
+    public void
+            itShouldReturn401OnUpdateEndorsementStatusWithOutSuperUserCookieAndAcceptOrRejectEndorsementStatus() {
+        String endorsementId = createEndorsementModel(false);
+        Response response =
+                given()
+                        .cookies(RestAPIHelper.getUserCookie())
+                        .queryParam("status", EndorsementStatus.APPROVED.name())
+                        .patch("/v1/endorsements/{id}", endorsementId);
+
+        response
+                .then()
+                .statusCode(401)
+                .body("data", equalTo(null))
+                .body("message", equalTo("Unauthorized, Access is only available to super users"));
```

Comment:
@Atifsid - Instead of the message in line 557, it should be "The access token provided is expired, revoked, malformed, or invalid for other reasons." according to https://github.com/Real-Dev-Squad/skill-tree-backend/blob/df3fbdd0dc55807676ee6966b87db608c83eb7da/skill-tree/src/main/java/com/RDS/skilltree/utils/GlobalExceptionHandler.java#L28.

Reply:
Yes @Atifsid but this is integration test, here https://github.com/Real-Dev-Squad/skill-tree-backend/blob/df3fbdd0dc55807676ee6966b87db608c83eb7da/skill-tree/src/main/java/com/RDS/skilltree/utils/GlobalExceptionHandler.java#L34 will override your message.

Reply:
Hi @Shaily20 
can we connect regarding this?